### DISCUSSION TITLE
Sending live video with socket.io

### DISCUSSION THREAD

**Discussion by crupeng-1** [socketio/socket.io#3901] [Q&A]
I want to send live video for a video chat app, I cannot use webRTC as it is not scalable enough for my needs. But socket.io seems too slow with my current implementation in which I send images from the webcam. Any suggestions to quickly send webcam frames with socket.io at a reasonable fps? (right now I can only manage 1fps)

**Comment by maxicarlos08** [comment]
What is wrong with WebRTC? WebSockets were not made to transfer videos
